### ISSUE TITLE
Long-term access to code and corpus repos

### ISSUE THREAD

**Issue by band** [Git-Lit/git-lit#60]
Github is not a repository or service that provides long-term preservation of content. See [Disclaimer of Warranties](https://help.github.com/articles/github-terms-of-service/#o-disclaimer-of-warranties). For long-term preservation of the code [Zenodo](https://zenodo.org/) supports preservation of Gihub repositories.

**Comment by JonathanReeve** [comment]
Good point. I've already done this for some TEI editions at https://github.com/open-editions, but not yet Git-Lit texts. 

Zenodo seems to require that we: 
 -  "flip the switch" on each repository individually in order to enable it
 -  create git releases for each repository we want to archive

The second should be relatively easy (script up the git commands), but the first would be harder. It might just require clicking something 50K times. But 50K repositories might overload Zenodo. Maybe Zenodo has an API that could handle this sort of thing? If you can figure out something, feel free to submit a PR.

**Comment by band** [comment]
@JonathanReeve Good point regarding size and frequency of posting to Zenodo.

I am not sure what a practicable approach is for the corpus of documents. Somehow actual documents should be posted to a preservation repository for long-term keeping. Github is a good workbench for developing tools, procedures, and work practices. I need to think about this a bit more.

The best use of Zenodo for this project is as a repository of milestone releases of the tools.
